modules/authentication/proc-enabling-authentication-with-microsoft-azure.adoc

```diff
@@ -0,0 +1,148 @@
+[id="enabling-authentication-with-microsoft-azure"]
+= Enabling authentication with Microsoft Azure
+
+{product} includes a Microsoft Azure authentication provider that can authenticate users by using OAuth.
+
+.Prerequisites
+. You have the permission to register an application in Microsoft Azure.
+. You link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/administration_guide_for_red_hat_developer_hub/index#assembly-add-custom-app-file-openshift_admin-rhdh[added a custom {product-short} application configuration].
+
+.Procedure
+. To allow {product-short} to authenticate with Microsoft Azure, create an OAuth Application in Microsoft Azure.
+
+.. To link: https://learn.microsoft.com/en-us/entra/identity-platform/scenario-web-app-sign-user-app-registration?tabs=aspnetcore#register-an-app-by-using-the-azure-portal[register the {product-short} application by using the Azure portal], go to link:https://portal.azure.com/#view/Microsoft_AAD_RegisteredApps/ApplicationsListBlade[*Azure Portal > Identity > Identity management > App registrations*] and click **New registration**.
```

Comment:
@JessicaJHee I need your help again

With my Red Hat account, this iswhat I see when I log in to Entra:

![Screenshot_2024-09-30_2](https://github.com/user-attachments/assets/dbacd452-af77-457e-9198-a0d50cf9de28)


I do not see how to do this sequence: *Azure Portal > Identity > Identity management > App registrations*  link:https://entra.azure.com/#view/Microsoft_AAD_RegisteredApps/ApplicationsListBlade  and click **New registration**.

I do not see how do to this sequence either: *Microsoft Entra ID > Manage > App registrations > click on + New registration*